com.typesafe.hocon.tests/src/com/typesafe/hocon/parser/ParserTest.xtend

```diff
@@ -118,7 +120,7 @@ class ParserTest {
 
   @Test
   def void testSubstitution() {
-    succeeds("${foo.bar} += ${akka.timeout}\n")
+    succeeds("foo.bar += ${akka.timeout}\n")
```

Comment:
This test was incorrect. HOCON does not allow a substitution on the left site of a declaration.
I check myself - a play app for example with such a statement would not start up and fail with with a parse exception.